pkg/devfile/parser/data/v2/common/component_helper.go

```diff
@@ -29,6 +29,8 @@ func GetComponentType(component v1.Component) (v1.ComponentType, error) {
 		return v1.KubernetesComponentType, nil
 	case component.Openshift != nil:
 		return v1.OpenshiftComponentType, nil
+	case component.Image != nil:
+		return v1.ImageComponentType, nil
 	case component.Custom != nil:
 		return v1.CustomComponentType, nil
 
```

Comment:
Do we need to add an IsImage() check for testing?

Reply:
If you check the file, we do not have this helper function for all component types, so I did not add `IsImage()`, and we can always add more helper functions if there is a request from consumers. 
I'm fine with adding `isImage` if you think it should be added

Reply:
Yes, we are missing that check for other components as well and I wasn't sure if it was an oversight or not.  I'm ok with not adding it if there is no immediate value in testing it.